docs/java/features/odata/overview.md

```diff
@@ -0,0 +1,33 @@
+---
+id: overview
+title: OData with the Cloud SDK for Java
```

Comment:
I don't see much that is only Java related in this overview. Either make it a general overview or emphasize Java features more. Generally, please never refer to the Java SDK as `SAP Cloud SDK` only in this shared documentation, use `SAP Cloud SDK for Java` instead. 
@artemkovalyov This should probably also be part of our style guide.


I think it is also fine if this is clarified somewhere in the beginning of the article. In the following whenever the SAP Cloud SDK is mentioned we refer to the SAP Cloud SDK for Java or something in that sense.